src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -111,6 +115,32 @@ private void testGeneralQuestionOperations() throws Exception {
         testResponseRate();
     }
 
+    private void testRankQuestionOperations() throws Exception {
+        testNewQuestionLink("RANK_RECIPIENTS");
+
+        ______TS("Empty Rank Options Message");
```

Comment:
What does message mean here? The status message or the error the browser is returning?

Can we change it to something clearer like `Invalid empty number of options a respondent must rank`?

Then add a comment on how it is covered for both min and max options.

Then also test `Invalid letters in number of options a respondent must rank`.

But with all that said, most of what you are testing is already tested in `FeedbackRankQuestionUiTest` but the invalid cases are not tested.

And anyhow, you need to modify the methods to make them work properly. For example,

```Java
    public void setMinOptionsToBeRanked(int qnNumber, int value) {
        assertTrue(isMinOptionsToBeRankedEnabled(qnNumber));

        WebElement inputBox = getMinOptionsToBeRankedInputElement(qnNumber);
        JavascriptExecutor exec = (JavascriptExecutor) browser.driver;
        String id = inputBox.getAttribute("id");

        exec.executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
    }
```

should really be just:
```Java
    public void fillMinOptionsToBeRankedSpecial(int qnNumber, String value) {
        WebElement inputElement = getMinOptionsToBeRankedInputElement(qnNumber);
        fillTextBox(inputElement, value);
    }
```

So if you want, can create a new issue to address the testing separately because this PR unfortunately touches too many parts of the system that are not properly designed.